lib/taste_tester/windows.rb

```diff
@@ -0,0 +1,64 @@
+# vim: syntax=ruby:expandtab:shiftwidth=2:softtabstop=2:tabstop=2
+
+# Copyright 2013-present Facebook
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module TasteTester
+  module Windows
+    PATH_TO_WRAPPER_SCRIPT =
+      File.absolute_path(File.join(
```

Comment:
OK, so if I understand this all correctly, the only reason for the wrapper script is so that when the scheduled task gets run, we shove the PID into the statefile?

That's... terrible. I mean it's a neat hack... but Jesus, Microsoft.